frontend/appflowy_tauri/src/appflowy_app/components/_shared/DatabaseFilter/DatabaseFilterItem.tsx

```diff
@@ -0,0 +1,411 @@
+              <div
+                onClick={() => setCurrentValue(!currentValue)}
+                className={`flex w-[180px] cursor-pointer items-center gap-2 rounded-lg border border-line-border px-2 py-1`}
+              >
+                <button className={'h-5 w-5'}>
+                  {currentValue ? <EditorCheckSvg></EditorCheckSvg> : <EditorUncheckSvg></EditorUncheckSvg>}
+                </button>
+                <span>{currentValue ? 'Checked' : 'Unchecked'}</span>
+              </div>
+            )}
+          </>
+        ) : (
+          <div className={`flex w-[180px] items-center justify-between rounded-lg border border-line-border px-2 py-1`}>
+            <span className={'text-text-placeholder'}>Select field</span>
+          </div>
+        )}
+
+        <button
+          onClick={() => onDelete?.()}
+          className={`rounded p-1 hover:bg-fill-list-hover ${data ? 'opacity-100' : 'opacity-0'}`}
+        >
+          <i className={'block h-[16px] w-[16px]'}>
+            <TrashSvg />
+          </i>
+        </button>
+      </div>
+
+      {showFieldSelect && (
+        <PopupSelect
+          items={supportedColumns.map<IPopupItem>((c) => ({
+            icon: (
+              <i className={'block h-5 w-5'}>
+                <FieldTypeIcon fieldType={fields[c.fieldId].fieldType}></FieldTypeIcon>
+              </i>
+            ),
+            title: fields[c.fieldId].title,
+            onClick: () => onSelectFieldClick(c.fieldId),
+          }))}
+          className={'fixed z-10 text-sm'}
+          style={{ top: `${fieldSelectTop}px`, left: `${fieldSelectLeft}px`, width: `${180}px` }}
+          onOutsideClick={() => setShowFieldSelect(false)}
+        ></PopupSelect>
+      )}
+
+      {showOperatorSelect && (
+        <PopupSelect
+          items={SupportedOperatorsByType[currentFieldType ? currentFieldType : FieldType.RichText].map<IPopupItem>(
+            (operatorName) => ({
+              icon: null,
+              title: operatorName,
+              onClick: () => onSelectOperatorClick(operatorName),
+            })
+          )}
+          className={'fixed z-10 text-sm'}
+          style={{ top: `${operatorSelectTop}px`, left: `${operatorSelectLeft}px`, width: `${180}px` }}
+          onOutsideClick={() => setShowOperatorSelect(false)}
+        ></PopupSelect>
+      )}
+
+      {showLogicalOperatorSelect && (
+        <PopupSelect
+          items={LogicalOperators.map<IPopupItem>((operatorName) => ({
+            icon: null,
+            title: operatorName,
+            onClick: () => onSelectLogicalOperatorClick(operatorName),
+          }))}
+          className={'fixed z-10 text-sm'}
+          style={{ top: `${logicalOperatorSelectTop}px`, left: `${logicalOperatorSelectLeft}px`, width: `${88}px` }}
+          onOutsideClick={() => setShowLogicalOperatorSelect(false)}
+        ></PopupSelect>
+      )}
+
+      {showValueOptions && currentFieldId && (
+        <PopupWindow
```

Comment:
Maybe you can use `import Popover from @mui/material`.  I unified the style of this part, you can use it directly

Reply:
Agree, changed to Popover